allennlp/data/dataset_readers/multiprocess_dataset_reader.py

```diff
@@ -30,26 +33,119 @@ def info(cls, message: str) -> None:
 def _worker(reader: DatasetReader,
             input_queue: Queue,
             output_queue: Queue,
-            index: int) -> None:
+            active_workers: Value,
+            inflight_items: Value,
```

Comment:
should these be called `num_active_workers` and `num_inflight_items` so that it's clearer what they're doing?

Reply:
Good call, done.

Reply:
Sorry about that missing that reference. Updated and grepped for any more stray instances. Should be good now.